From 31f27edda12737b33e73b58df0889067e86653f9 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Fri, 14 Jun 2024 14:23:02 -0700
Subject: [PATCH] ==== damos_migrate docs fixup ====

Subject: fixup DAMOS_MIGRATE_{HOT,COLD} patchset documentation

DAMOS_MIGRATE_{HOT,COLD} documentation[1], which is currently merged in
the mm-unstable tree, has two trivial problems.

Firstly, the usage document.  It missed updating some parts of the
document, and the layout of some paragraphs could be more consistent.
Secondly, DAMON sysfs ABI document is not updated.  Fix the two trivial
problems with two patches, respectively.

Andrew, could you plaese squash the first patch for usage document fixup
into the original patch[1]?  Meanwhile, I'd prefer keeping the second
patch as separate one, since I prefer having individual commit to touch
single file if possible.

[1] c100fc71858a ("Docs/damon: document damos_migrate_{hot,cold}") # mm-unstable

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/Gk4zsfJ9 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/Gk4zsfJ9

diff --git a/damon_meta_changes/Gk4zsfJ9 b/damon_meta_changes/Gk4zsfJ9
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

